js/irma-jwt/package-lock.json

```diff
@@ -1,27 +1,35 @@
 {
   "name": "@privacybydesign/irma-jwt",
   "version": "0.1.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
   "requires": true,
-  "dependencies": {
-    "buffer-equal-constant-time": {
+  "packages": {
+    "": {
+      "name": "@privacybydesign/irma-jwt",
+      "version": "0.1.1",
+      "license": "MIT",
+      "dependencies": {
+        "jsonwebtoken": "^8.5.1"
+      }
+    },
+    "node_modules/buffer-equal-constant-time": {
       "version": "1.0.1",
       "resolved": "https://registry.npmjs.org/buffer-equal-constant-time/-/buffer-equal-constant-time-1.0.1.tgz",
-      "integrity": "sha1-+OcRMvf/5uAaXJaXpMbz5I1cyBk="
+      "integrity": "sha512-zRpUiDwd/xk6ADqPMATG8vc9VPrkck7T07OIx0gnjmJAnHnTVXNQG3vfvWNuiZIkwu9KrKdA1iJKfsfTVxE6NA=="
     },
-    "ecdsa-sig-formatter": {
+    "node_modules/ecdsa-sig-formatter": {
       "version": "1.0.11",
       "resolved": "https://registry.npmjs.org/ecdsa-sig-formatter/-/ecdsa-sig-formatter-1.0.11.tgz",
       "integrity": "sha512-nagl3RYrbNv6kQkeJIpt6NJZy8twLB/2vtz6yN9Z4vRKHN4/QZJIEbqohALSgwKdnksuY3k5Addp5lg8sVoVcQ==",
-      "requires": {
+      "dependencies": {
         "safe-buffer": "^5.0.1"
       }
     },
-    "jsonwebtoken": {
+    "node_modules/jsonwebtoken": {
       "version": "8.5.1",
```

Comment:
I think you somehow added the wrong `package-lock.json` because this version here doesn't correspond with the version in `package.json`.

Reply:
When updating the package it apparently did not update the package-lock.json, now after regenerating, it is updated